### ISSUE TITLE
dont set pip module executable by default

### ISSUE THREAD

**Issue by kzap** [geerlingguy/ansible-role-pip#17]
If your playbook sets the python interpreter to python3 like
```yaml
ansible_python_interpreter: /usr/bin/python3
```

and then you use this role without setting
```yaml
pip_package: python3-pip
```

It will use `pip` by default:
https://github.com/geerlingguy/ansible-role-pip/blob/0717b46dc20e6829c20158875c29427a3b81f8bc/tasks/main.yml#L13

imho I feel it would be better not to set a default executable and omit it unless one is specifically set since the `pip` module will then use whatever python interpreter ansible is using.

in short i do not agree the `executable` should have a default value :)
